sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -112,6 +112,7 @@ export async function saveFakeApplicationDisbursements(
   },
   options?: {
     applicationStatus?: ApplicationStatus;
+    updatedAt?: Date;
```

Comment:
if updatedAt needs to modified for test data preparation of one test suite, then it can be set at the consumer level i.e. the test(s) which need it rather than updating the factory.